src/main/kotlin/racing/service/GameMachine.kt

```diff
@@ -0,0 +1,32 @@
+package racing.service
+
+import racing.model.Racing
+
+object GameMachine {
+    private const val CAR_SYMBOL = "-"
+    private const val RESULT_SENTENCE = "실행 결과"
+    fun askCounting(question: String): Int {
```

Comment:
적절한 줄바꿈을 통해서 프로퍼티와 함수를 구분하기 쉽게 분리해주시면 좋을 것 같아요. 😃
```suggestion
    private const val RESULT_SENTENCE = "실행 결과"
    
    fun askCounting(question: String): Int {
```